src/plugins/save-refresh.js

```diff
@@ -0,0 +1,39 @@
+module.exports = bookshelf => {
+  // model.save() wouldn't return all columns
+  // https://github.com/bookshelf/bookshelf/issues/507
+  // TODO: this feature may be supported after bookshelf v0.14.x
+  // https://github.com/bookshelf/bookshelf/issues/1665
+  const proto = bookshelf.Model.prototype;
+
+  // eslint-disable-next-line no-param-reassign
+  bookshelf.Model = bookshelf.Model.extend({
+    async save(...args) {
+      let attrs;
+      let options;
+
+      const [firstArg, secondArg, thirdArg] = args;
+
+      // Handle both `"key", value` and `{key: value}` -style arguments.
+      if (firstArg == null || typeof firstArg === 'object') {
+        attrs = firstArg || {};
+        options = secondArg || {};
+      } else {
+        attrs = {
+          [firstArg]: secondArg,
+        };
+        options = thirdArg || {};
+      }
```

Comment:
Shell we just get `withRefresh` in options from arguments ?
`attrs` are already handled by native saved method

Reply:
But we need to get options from 2 or 3:  

```
user.save({ key: value }, { withRefresh: true });
user.save(key, value, { withRefresh: true });
```

Reply:
:confused: 
OK, that's fine.